synapse/handlers/sync.py

```diff
@@ -1717,7 +1715,7 @@ async def _generate_sync_entry_for_to_device(
 
     async def _generate_sync_entry_for_account_data(
         self, sync_result_builder: "SyncResultBuilder"
-    ) -> Dict[str, Dict[str, JsonDict]]:
+    ) -> None:
```

Comment:
This function's docstring is out of date: it refers to a return value, but it has been changed to `None`.

Reply:
I think this function is now only handling the global account data?